### PR TITLE
Security front and backend

### PR DESCRIPTION
**Pull request by abnaemi**
code smells will be removed later in polishing phase

### CODE REVIEW

**Comment by abnaemi** [comment]
code smells will be removed in a later phase, as i am unable to do right now.
sorry for the long pr. if to much, just review it superficial

File: backend/src/main/java/de/neuefische/backend/security/SecurityConfig.java
Comment (junesdream) [inline backend/src/main/java/de/neuefische/backend/security/SecurityConfig.java:1]: Line

File: backend/src/test/java/de/neuefische/backend/ticket/TicketIntegrationTest.java
Comment (junesdream) [inline backend/src/test/java/de/neuefische/backend/ticket/TicketIntegrationTest.java:1]: Long

File: frontend/package-lock.json
Comment (junesdream) [inline frontend/package-lock.json:1]: Unless

**Review by junesdream — APPROVED** [review]

**Pull request merged**